datastore/src/main/java/com/ds/datastore/BookController.java

```diff
@@ -63,10 +71,10 @@ protected Book updateBook(@RequestBody Book newBook, @PathVariable Long id) {
     }
 
     @ResponseStatus(HttpStatus.NO_CONTENT)
-    @DeleteMapping("/books/{id}")
-    protected void deleteBook(@PathVariable Long id) {
+    @DeleteMapping("/bookstores/{storeID}/books/{id}")
+    protected void deleteBook(@PathVariable Long id, @PathVariable Long storeID) {
         repository.findById(id).orElseThrow(() -> new BookNotFoundException(id));
         repository.deleteById(id);
     }
 
-}
+}
```

Comment:
this is a small deal, and it bothers me more than it should :), but lets add a newline to the end of every file. That is kind of convention.